boxoffice/views/order.py

```diff
@@ -26,13 +28,16 @@ def jsonify_line_items(line_items):
     """
     items_json = dict()
     for line_item in line_items:
+        item = Item.query.get(line_item.item_id)
+        quantity_available = item.quantity_total - item.get_confirmed_line_items.count()
```

Comment:
`item.quantity_available` was originally introduced with the intention of letting an admin manually update the value. But now, it seem simpler to only have `item.quantity_total` and compute the availability.